governance/constitution/actions/default.js

```diff
@@ -0,0 +1,1481 @@
+class Action {
```

Comment:
is this the default action as it is defined in CCF?
If so,  why do we redefine it. Our current set_constitution just specified the delta we define in KMS.

Reply:
Because baking the assumption into the script that we will add stuff on top of the sandbox default constitution makes changing things difficult. (What if we want a different resolve function?)

Reply:
you can add just your delta's to the default constitution. That is what I did with kms_sctions.js.
Now you will have to maintain the constitution, changed in CCF have to be copied.

Reply:
The existing approach is fine if you're just appending to the existing constitution, but if you want to change something in it (you suggested many possible changes, I've implemented different resolve functions as a concrete example) then your approach still requires committed code, likely a patch? 

If that code is indeed a patch against assumed defaults, it will need maintenance if the default constitution changes, in fact your suggested approach would break whereas the one in this PR can be updated if we choose but will continue to work if we don't